app/web/features/auth/signup/ResendVerificationEmailForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { Typography } from "@material-ui/core";
+import Alert from "components/Alert";
+import StyledLink from "components/StyledLink";
+import { useAuthContext } from "features/auth/AuthProvider";
+import { RpcError } from "grpc-web";
+import { Trans, useTranslation } from "i18n";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+import { useState } from "react";
+import { useMutation } from "react-query";
+import { service } from "service";
+
+export default function ResendVerificationEmailForm() {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+  const { authActions, authState } = useAuthContext();
+
+  const [resent, setResent] = useState<boolean>(false);
+
+  const mutation = useMutation<void, RpcError>(
+    async () => {
+      const state = await service.auth.signupFlowResendVerificationEmail(
+        authState.flowState!.flowToken
+      );
+      authActions.updateSignupState(state);
+      setResent(true);
+    },
+    {
+      onMutate() {
+        authActions.clearError();
```

Comment:
why clear error here? (just out of curiosity)

Reply:
I guess it'll set the error during the mutation if there's an issue, since it's async